model/message_broker.go

```diff
@@ -0,0 +1,164 @@
+package model
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/evergreen-ci/cedar"
+	"github.com/mongodb/anser/bsonutil"
+	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/message"
+	"github.com/mongodb/grip/recovery"
+	"github.com/pkg/errors"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+const topicsCollection = "topics"
+
+// MessageEntry represents an entry for a "message" of a topic in the database.
+type MessageEntry struct {
+	// Topic is the name of the topic.
+	Topic string `bson:"topic"`
+	// ForeignID is the id of a document located in another collection used
+	// by the Topic interface to extract the actual message. The id of each
+	// message entry is an int sequentially incremented for each message
```

Comment:
where do we set the ID of the message entries? Why do we want an increasing int id?

Reply:
That is an outdated comment, going to update it.